app/presenters/two_factor_authentication/webauthn_platform_selection_presenter.rb

```diff
@@ -17,10 +17,6 @@ def render_in(view_context, &block)
       )
     end
 
-    def disabled?
-      user&.webauthn_configurations&.where(platform_authenticator: true)&.any?
-    end
-
     def mfa_configuration_count
```

Comment:
Per [discussion in Slack](https://gsa-tts.slack.com/archives/C01710KMYUB/p1691669528559429?thread_ts=1691616397.404099&cid=C01710KMYUB), I think we landed on wanting F/T to be restricted to single-only during account creation.

```suggestion
    def single_configuration_only?
      true
    end

    def mfa_configuration_count
```

Reply:
That is added in and the test for the presenter is updated also.